### PR TITLE
feat(components): [switch] Add custom activeActionIcon and inactiveActionIcon slots

### PR DESCRIPTION
**Pull request by xing403**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

## Description
Add custom activeActionIcon and inactiveActionIcon slots to switch component

## Related Issue

Fixes #\_\_\_.

### CODE REVIEW

**Comment by btea** [comment]
It looks similar to this [feature](https://element-plus.org/zh-CN/component/switch.html#%E8%87%AA%E5%AE%9A%E4%B9%89%E5%8A%A8%E4%BD%9C%E5%9B%BE%E6%A0%87).

**Comment by xing403** [comment]
This feature may be different, using slot can be more flexible to use 'icon' or other custom `icon`

File: packages/components/switch/src/switch.vue [hunk 1]
```diff
@@ -45,12 +45,18 @@
...
+          <el-icon v-if="activeActionIcon || $slots.activeActionIcon">
+            <component :is="activeActionIcon" v-if="activeActionIcon" />
+            <slot v-else name="activeActionIcon" />
+          </el-icon>
+        </template>
+        <template v-else-if="!checked">
+          <el-icon v-if="inactiveActionIcon || $slots.inactiveActionIcon">
+            <component :is="inactiveActionIcon" v-if="inactiveActionIcon" />
+            <slot v-else name="inactiveActionIcon" />
+          </el-icon>
```
Comment (kooriookami) [inline packages/components/switch/src/switch.vue:58]: This looks weird, there're two icons here.

Comment (xing403) [reply packages/components/switch/src/switch.vue:58]: Here are two ways: one is to display ICONS using attributes, the other is to display ICONS using slots, and I have tested that the use of these two ways should not display two ICONS at the same time

File: packages/components/switch/src/switch.vue [hunk 2]
```diff
@@ -45,12 +45,18 @@
...
+        </template>
```
Comment (kooriookami) [inline packages/components/switch/src/switch.vue:59]:
```suggestion
        <slot v-else-if="checked" name="activeActionIcon">
          <el-icon v-if="activeActionIcon">
            <component :is="activeActionIcon" />
          </el-icon>
        </slot>
        <slot v-else-if="!checked" name="inactiveActionIcon">
          <el-icon v-if="inactiveActionIcon">
            <component :is="inactiveActionIcon" />
          </el-icon>
        </slot>
```
Sometimes users want to add other things like words or images in slots. @btea What do you think?

**Review by kooriookami — CHANGES_REQUESTED** [review]

File: packages/components/switch/src/switch.vue (hunk 2 above)
Comment (xing403) [reply packages/components/switch/src/switch.vue]: The code has been modified as suggested

**Comment by kooriookami** [comment]
@xing403 Please don't upload `pnpm-lock.yaml`, the test failed.

**Comment by xing403** [comment]
I'm sorry. :flushed:

**Comment by kooriookami** [comment]
The lock file is still exist.

**Pull request closed without merging**